packages/components/src/progress-bar/styles.ts

```diff
@@ -43,7 +49,16 @@ export const Indicator = styled.div< {
 	top: 0;
 	height: 100%;
 	border-radius: ${ CONFIG.radiusBlockUi };
-	background-color: ${ COLORS.theme.accent };
+	/* Text color at 90% opacity */
+	background-color: color-mix(
+		in srgb,
+		var( --wp-components-color-foreground, ${ COLORS.gray[ 900 ] } ),
```

Comment:
Do we mean to default both indicator and track color to the same shade of gray? I'd expect the track and indicator fallback colors to be different from each other.

Reply:
This PR is implementing [this suggestion](https://github.com/WordPress/gutenberg/issues/54202#issuecomment-1729003302) from Joen. I think that it makes sense to use the same shade of gray and play with opacity, because it ensures high contrast against the background.